decidim-core/spec/helpers/decidim/sanitize_helper_spec.rb

```diff
@@ -35,6 +39,9 @@ module Decidim
           expect(helper.decidim_sanitize(user_input)).to include("<p>")
           expect(helper.decidim_sanitize(user_input)).to include("</p>")
           expect(helper.decidim_sanitize(user_input)).to eq(user_input)
+          expect(helper.decidim_sanitize_newsletter(user_input)).to include("<p>")
```

Comment:
These tests are a litte wwird because they're testing two different methods, I think it's probably better to have different specs! 😄